test/sanity/pylint/ignore.txt

```diff
@@ -1 +1,64 @@
+lib/ansible/module_utils/network/iosxr/iosxr.py ansible-format-automatic-specification
```

Comment:
These ignores are not related to this PR and should be reverted.

Reply:
@mattclay done! sorry i resolved de conflict badly